en-US/BoardComparison.md

```diff
@@ -147,7 +147,7 @@ lang: en-US
         <p>1x SPI bus</p>
         <p>1x I2C bus</p>
         <p>2x Serial UARTs</p>
-        <p><a href="{{site.baseurl}}/{{page.lang}}/win10/MBM.htm">Set up your MinnowBoard MAX</a></p>
```

Comment:
The others are already pointing to get started page. Only MBM was pointing to a different page.